src/helm/proxy/clients/palmyra_client.py

```diff
@@ -0,0 +1,129 @@
+import json
+import requests
+from typing import Any, Dict, List
+
+from helm.common.cache import Cache, CacheConfig
+from helm.common.request import Request, RequestResult, Sequence, Token
+from helm.common.tokenization_request import (
+    DecodeRequest,
+    DecodeRequestResult,
+    TokenizationRequest,
+    TokenizationRequestResult,
+)
+from .client import Client, wrap_request_time, truncate_sequence
+
+
+class PalmyraClient(Client):
+    def __init__(self, api_key: str, cache_config: CacheConfig, tokenizer_client: Client):
+        self.api_key: str = api_key
+        self.cache = Cache(cache_config)
+        self.tokenizer_client = tokenizer_client
+
+    def _send_request(self, model_name: str, raw_request: Dict[str, Any]) -> Dict[str, Any]:
+        response = requests.request(
+            method="POST",
+            url=f"https://enterprise-api.writer.com/llm/organization/3002/model/{model_name}/completions",
+            headers={
+                "Content-Type": "application/json",
+                "Accept": "application/json",
+                "Authorization": f"Bearer {self.api_key}",
+            },
+            data=json.dumps(raw_request),
+        )
+        result = json.loads(response.text)
+        if "error" in result:
+            raise ValueError(f"Request failed with error: {result['error']}")
+        return result
+
+    def make_request(self, request: Request) -> RequestResult:
+        """Make a request"""
+        raw_request = {
+            "prompt": request.prompt,
+            "maxTokens": request.max_tokens,
+            "temperature": request.temperature,
+            "topP": request.top_p,
+            "bestOf": request.top_k_per_token,
+            "stop": request.stop_sequences,
+            "random_seed": request.random,
```

Comment:
The [API docs](https://dev.writer.com/reference/completions-1) doesn't have a random_seed parameter, is this an undocumented parameter, or is this silently ignored by the API?

Reply:
Empirically:

- The requests and responses are cached on the API server-side, so making the same request will result in the same response, even if the temperature is high
- The value of `random_seed` is ignored; the same result is returned regardless of whether it is set and its value

I think we should remove `random_seed` until it is working.